src/serverdlg.cpp

```diff
@@ -184,10 +184,10 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
 
         pSystemTrayIconMenu->addSeparator();
 
-        pSystemTrayIconMenu->addAction ( tr ( "&Hide " ) + APP_NAME + tr ( " server" ), this, SLOT ( OnSysTrayMenuHide() ) );
+        pSystemTrayIconMenu->addAction ( tr ( "&Hide %1 server" ).arg ( APP_NAME ), this, SLOT ( OnSysTrayMenuHide() ) );
 
         pSystemTrayIconMenu->setDefaultAction (
-            pSystemTrayIconMenu->addAction ( tr ( "&Open " ) + APP_NAME + tr ( " server" ), this, SLOT ( OnSysTrayMenuOpen() ) ) );
+            pSystemTrayIconMenu->addAction ( tr ( "&Open %1 server" ).arg ( APP_NAME ), this, SLOT ( OnSysTrayMenuOpen() ) ) );
```

Comment:
As it's changing, can we make this "Show" rather than "Open"?

Reply:
Since this patchset is intended to fix the localization issues than tweaking the existing copywriting, I'm not very sure if we should change it in this patchset...

Anyway, if we change it, is this correct?

```suggestion
            pSystemTrayIconMenu->addAction ( tr ( "&Show %1 server" ).arg ( APP_NAME ), this, SLOT ( OnSysTrayMenuOpen() ) ) );
```

It will change the shortcut from `&O` (<kbd>Shift+O</kbd>) to `&S`(<kbd>Shift+S</kbd>).

Reply:
That's what I was thinking and, so long as the menu that pops up has no other &S shortcut, it's better to change it, yes.

Reply:
Done :)